CONTRIBUTING.md

```diff
@@ -10,57 +10,80 @@ Have a question or feature request? Rather than opening an issue, use the [Discu
 
 Please be polite and take the time to write a well-worded question so our community members can easily assist you.
 
-## Getting started with bug fixing
+## Prerequisites
 
-In order to make it easier to get familiar with the codebase we labeled simpler issues using [Good First Issue](https://github.com/parcel-bundler/parcel/issues?q=is%3Aopen+is%3Aissue+label%3A%22%E2%9C%A8+Parcel+2%22+label%3A%22%3Ababy%3A+Good+First+Issue%22) and [Help Wanted](https://github.com/parcel-bundler/parcel/issues?q=is%3Aopen+is%3Aissue+label%3A%22%E2%9C%A8+Parcel+2%22+label%3A%22%3Apray%3A+Help+Wanted%22).
+Before starting make sure you have the following installed:
 
-Before starting make sure you have the following requirements installed: [git](https://git-scm.com), [Node](https://nodejs.org), [Yarn](https://yarnpkg.com) and [Rust](https://www.rust-lang.org/tools/install).
+- [git](https://git-scm.com)
+- [Node](https://nodejs.org) at LTS
+- [Yarn](https://yarnpkg.com) at v1
+- [Rust](https://www.rust-lang.org/tools/install) stable
+- [Flow](https://flow.org/en/docs/editors) IDE autocompletion and type-checking
```

Comment:
You don't have to install flow separately though, it's installed via a npm dependency?

Reply:
It's the IDE editors mentioned here, it was previously mentioned in the file as

> Parcel uses [Flow](https://flow.org/) for type checking. If you're using an IDE, make sure to install the [Flow extension](https://flow.org/en/docs/editors/) to ensure your editor's autocomplete and type-checking works as expected.

Let me know if you still want it there, the wording should be updated, or removed entirely